### ISSUE TITLE
[Accessibility][Sourcing]: arrow keys not functioning properly on table with frozen columns

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9134]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
Wrong behaviour of left or right arrow keys when on a table with frozen columns

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 13.x.x
   "@fundamental-ngx/core": "0.35.4",
    "@fundamental-ngx/platform": "0.35.4",

#### If this is a bug, please provide steps for reproducing it.
1) Reach a fdp-table with one or two frozen columns
2) Try using left or right arrow on any of the rows

Actual: The frozen columns get scrolled left or right
Expected: The user should be able to traverse through different cells of that row

#### Please provide relevant source code if applicable.
Replicable on https://sap.github.io/fundamental-ngx/#/platform/table :Column Freezing section

#### Is there anything else we should know?
This is a blocker for guided sourcing customers

**Comment by I543348** [comment]
The visible focus is splitting across 2 columns , as per accessibility guidelines, the visible focus should be clearly visible .
This needs to be reopened
please see attached screenshot, 
<img width="1792" alt="Screenshot 2023-03-06 at 9 57 23 PM" src="https://user-images.githubusercontent.com/88182358/223171162-f8ac7b9b-6bbd-4f0f-9df7-827fbe9260cb.png">
